packages/loader/index.js

```diff
@@ -1,11 +1,19 @@
 const {getOptions} = require('loader-utils')
 const mdx = require('@mdx-js/mdx')
 
-module.exports = async function(content) {
+const DEFAULT_PREPEND = `
+  import React from 'react'
+  import { mdx } from '@mdx-js/react'
+`
+  .split('\n')
```

Comment:
In order to avoid this logic maybe we can dedent the template string?

```js
const DEFAULT_PREPEND = `
import React from 'react'
import { mdx } from '@mdx-js/react'
`
```